tests/serde-de.rs

```diff
@@ -512,6 +512,2530 @@ mod tuple_struct {
     }
 }
 
+mod seq {
+    use super::*;
+
+    /// Check that top-level sequences can be deserialized from the multi-root XML documents
+    mod top_level {
+        use super::*;
+        use pretty_assertions::assert_eq;
+
+        #[test]
+        fn simple() {
+            from_str::<[(); 3]>("<root/><root>42</root><root>answer</root>").unwrap();
```

Comment:
This will panic if it cannot be deserialized to an array of size 3 yes?  (I am not as familiar with serde, my code doesn't use it)

Just curious why it's not asserted like the Vec scenario, I'm assuming this is why.

Reply:
It's fine